src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -377,7 +377,7 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
 
                 if (recursivePatternType != null)
                 {
-                    return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpaces);
+                    return AdjustSpacesOperationZeroOrOne(optionSet, CSharpFormattingOptions.SpaceAfterMethodCallName);
```

Comment:
the comment above seems like it is no longer valid.  maybe it should be `update the space between a type and a positional pattern`?

Reply:
`CSharpFormattingOptions.SpaceAfterMethodCallName`

It's unclear to me why this would be the right option to use.  For one thing, this is not really a method-call, it's a positional pattern.  For another, the syntax is `Type DeconstructionPattern`.  So, it's not really the name of a method, it's the name of a type.  And, importantly, we don't use this formatting option to separate out a Type+List when you have `new Foo(...)`.  So i don't see why we would separate it out now that you have `Foo (...)` in a pattern.

Reply:
Objection retracted.  Something was messed up with my VS. Even though i enabled this option, it didn't add a space.  However, closing/reopening caused it to start adding the space.  So i agree with this change.

Reply:
Fixed!